website/docs/sync.md

```diff
@@ -36,11 +59,22 @@ You can install this config with the following command:
 kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/master/demo/basic/sync.yaml
 ```
 
-Once data is synced into OPA, rules can access the cached data under the `data.inventory` document.
+### Working with SyncSet and Config resources
```

Comment:
IMO, moving these within their own sections after `Replicating Data with SyncSets` and `Replicating Data with Config` respectively might be more clear.

Reply:
Hm, agreed but aren't those already their own section? As in they are on the same level as the `Replicating Data with *` sections atm. Everything is under `Replicating Data` as the title. Or did I misunderstand the question?

Reply:
I meant move “how to work with syncset” under “Replicating Data with Syncset” and same for configmap, instead of having both of the “how to” combined in the same section.

Reply:
Oh yes -- I adjusted the scenario.